### ISSUE TITLE
zero-byte file in d2l.zip

### ISSUE THREAD

**Issue by xiaotinghe** [d2l-ai/d2l-en#1681]
Tensorflow and Torch implementations in the d2l.zip contain some 0-byte files

- chapter_convolutional-neural-networks/why-conv.ipynb
- chapter_convolutional-neural-networks/index.ipynb
- chapter_linear-networks/softmax-regression.ipynb
- chapter_linear-networks/index.ipynb
- ...

**Comment by AnirudhDagar** [comment]
I'll try to see what can be done here in d2lbook when I work on the new infra soon.
